uptrain/constants.py

```diff
@@ -37,8 +38,9 @@ class Statistic(str, Enum):
 
 
 class Visual(str, Enum):
-    UMAP = "umap"
-    TSNE = "tsne"
+    UMAP = "UMAP"
+    TSNE = "t-SNE"
+    SHAP = "SHAP"
```

Comment:
Should we add SHAP to visuals?

Reply:
Not sure what's the best place. For now, it's more about visualization, so added it here.